src/main/java/com/rakovets/course/java/core/practice/oop/principles/cat_homework/Person.java

```diff
@@ -0,0 +1,44 @@
+package com.rakovets.course.java.core.practice.oop.principles.cat_homework;
+
+public class Person {
+    private int happiness;
+
+    public Person(int happiness) {
+        if (happiness>= 0 && happiness<= 100) {
+            this.happiness = happiness;
+        }else if (happiness < 0) {
+            this.happiness = 0;
+        }else if (happiness > 100) {
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"